src/lib/theme.ts

```diff
@@ -0,0 +1,10 @@
+import resolveConfig from "tailwindcss/resolveConfig";
+
+import tailwindConfig from "../../tailwind.config";
+
+
+const { theme } = resolveConfig(tailwindConfig);
+export const themeColors = theme?.colors as any as Record<
```

Comment:
Conversion of type 'DefaultColors & { border: string; input: string; ring: string; background: string; foreground: string; primary: { DEFAULT: string; foreground: string; }; secondary: { DEFAULT: string; foreground: string; }; ... 10 more ...; browseSwipe: { ...; }; }' to type 'Record<string, Record<string, string>>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Index signature for type 'string' is missing in type 'DefaultColors & { border: string; input: string; ring: string; background: string; foreground: string; primary: { DEFAULT: string; foreground: string; }; secondary: { DEFAULT: string; foreground: string; }; ... 10 more ...; browseSwipe: { ...; }; }'.ts(2352)
  
  this is an error message without as any before as Record